modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/utils/validations.form.js

```diff
@@ -40,6 +41,24 @@ const isLowercaseAndNumbers = (value) => {
 	}
 };
 
+const isStartingWithSlash = (value) => {
+	if (value[0] !== '/') {
+		return 'The Workspace URL should start with "/".';
+	}
+};
+
+const hasNoEmptySpace = (value) => {
+	if (value.indexOf(' ') > 0) {
+		return 'The Workspace URL most not have spaces.';
+	}
+};
+
+const hasFriendlyURL = (value) => {
+	if (!FRIENDLY_URL_REGEX.test(value)) {
+		return 'Lowercase letters, numbers and dashes only.';
+	}
+};
+
```

Comment:
```suggestion
Should be good centralize all these function within one, because are used them to same purpose

const isValidFriendlyURL = (value) => {
	if (value[0] !== '/') {
		return 'The Workspace URL should start with "/".';
	}
	
	if (value.indexOf(' ') > 0) {
		return 'The Workspace URL most not have spaces.';
	}
	
	if (!FRIENDLY_URL_REGEX.test(value)) {
		return 'Lowercase letters, numbers and dashes only.';
	}
};
```

Reply:
Good sugestions, Jair! Already done, thank you!!